.travis.yml

```diff
@@ -1,16 +1,38 @@
-language: python
-python:
-  - "3.6"
-cache: pip
-services:
-  - mysql
-before_install:
-  - mysql -e 'CREATE DATABASE IF NOT EXISTS pytest_testdb;'
-install:
-  - pip install -r requirements.txt
-before_script:
-  - black --check .
-script:
-  - flake8 . --count --select=E101,E722,E9,F4,F63,F7,F82,W191 --show-source --statistics
-  - flake8 . --count --exit-zero --max-line-length=127 --statistics
-  - pytest -v
+#----------linux 16.04------------ 
+  language: python
+  os: linux
+  dist: xenial
+  python:
+    - 3.6
+  cache: pip
+  services:
+    - mysql
+  before_install:
+    - mysql -e 'CREATE DATABASE IF NOT EXISTS pytest_testdb;'
+    

```

Comment:
Why have the Linux jobs been removed?

Reply:
Including them in the job matrix gave an error, I refered to a few repositories and this is the style they followed. Is it not checking for Linux?

Reply:
No, it's only running for osX